simulators/portal-interop/src/main.rs

```diff
@@ -641,33 +641,53 @@ dyn_async! {
             }
         }
 
-        // wait 8 seconds for data to propagate
-        // This value is determined by how long the sleeps are in the gossip code
-        // So we may lower this or remove it depending on what we find.
-        tokio::time::sleep(Duration::from_secs(8)).await;
-
-        let comments = vec!["1 header", "1 block body", "100 header",
-            "100 block body", "7000000 header", "7000000 block body",
-            "7000000 receipt", "15600000 (post-merge) header", "15600000 (post-merge) block body", "15600000 (post-merge) receipt",
-            "17510000 (post-shanghai) header", "17510000 (post-shanghai) block body", "17510000 (post-shanghai) receipt"];
-
+        // wait content_vec.len() seconds for data to propagate, giving more time if more items are propagating
```

Comment:
`// wait test_data.len() seconds ... `

Reply:
I am not sure what you are trying to imply?

Reply:
If I'm not mistaken, the comment is incorrect. We are using `test_data.len()` in the line below to figure out how long the wait should be, so the comment should be updated with `test_data.len()` instead of `content_vec.len()`

Reply:
fixed it